include/libtorrent/torrent_info.hpp

```diff
@@ -111,6 +111,20 @@ namespace libtorrent {
 	// used to disambiguate a bencoded buffer and a filename
 	extern TORRENT_EXPORT from_span_t from_span;
 
+	// this object holds configuration options for limits to use when loading
+	// torrents. They are meant to prevent loading potentially malicious torrents
+	// that cause excessive memory allocations.
+	struct load_torrent_limits
+	{
+		int max_buffer_size = 6000000;
```

Comment:
Is this supposed to be the file size limit in bytes?

Reply:
yes. the .torrent-file size limit